src/lib/core/hooks/checkedLegendItemsStatus.ts

```diff
@@ -15,3 +15,52 @@ export function useCheckedLegendItemsStatus(
 
   return checkedLegendItems;
 }
+
+/**
+ * WIP custom hook to wrap all necessary checked legend items functionality
+ * into one
+ */
+
+export function useCheckedLegendItems(
```

Comment:
This doesn't seem used, but I think that was the plan (to wait until after the mega merge)